src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectOfStyles = {};
+  const arrayOfStyles = splitAndTrim(sourceString, ';')
+    .filter(style => style.length > 0);
+  const splitedStyles = arrayOfStyles.map(style => splitAndTrim(style, ':'));
+
+  splitedStyles.map(function makeObj(style) {
+    objectOfStyles[style[0]] = style[1];
+  });
```

Comment:
You don't need to use `map` here.
Map return new array. You can use `forEach` instead `map` in this case.

Use arrow functions for callbacks inside methods.